app/models/enrollment.rb

```diff
@@ -0,0 +1,10 @@
+class Enrollment < ActiveRecord::Base
+  belongs_to :game
+  belongs_to :white_player_id, class_name: "Player"
+  belongs_to :black_player_id, class_name: "Player"
+
+  def join_game(game)
```

Comment:
`join_game!`

Although this should be in the game model and simply be `join!`. Think:
* If I am in ModelA and only modifying one of related ModelB, I should be in ModelB
* If I am in ModelA and am potentially modifying a collection of related ModelB, I should be in ModelA